StorageUtility/Linux/StorageUtility.cpp

```diff
@@ -17,251 +17,181 @@ limitations under the License.
 */
 
 #include <libudev.h>
+#include <fcntl.h>
+#include <map>
 #include <memory>
 #include <string.h>
+#include <unistd.h>
 
-#include "StorageUtility.h"
 #include "Device.h"
+#include "StorageUtility.h"
 
 namespace vtStor
 {
 
-eErrorCode GetStorageDevices(std::vector<std::shared_ptr<cDeviceInterface>>& Devices, eOnErrorBehavior OnErrorBehavior)
+void GetStorageDevices(std::vector<std::shared_ptr<cDeviceInterface>>& Devices, eOnErrorBehavior OnErrorBehavior)
```

Comment:
We should have ways to return which errorcode in this function, you are using a function GetStorageDevicePaths which return errorcode but not use in this section